content/applications/hr/payroll/work_entries.rst

```diff
@@ -0,0 +1,186 @@
+============
+Work Entries
+============
+
+The :guilabel:`Work Entries` dashboard, which can be found by going to :menuselection:`Payroll
+--> Work Entries --> Work Entries`, provides a visual overview of the individual time sheets of
+every employee, with each day split into a morning shift and an afternoon shift.
+
+.. image:: work_entries/work-entries-overview.png
+   :align: center
+   :alt: Work Entries dashboard view showing all employee's work entries.
+
+To change the view so that only the entries for a single day, week, or month are shown, click on
+either :guilabel:`Day`, :guilabel:`Week`, or :guilabel:`Month`.
```

Comment:
wording (so less redundant) + add location of the clickable objects mentioned in the instructions.

```suggestion
one of the corresponding links for :guilabel:`Day`, :guilabel:`Week`, or :guilabel:`Month`, located at the top of the dashboard.
```